webpack.config.js

```diff
@@ -331,6 +331,21 @@ function getWebpackConfig( {
 		);
 	}
 
+	// List of polyfills that we skip including in the evergreen bundle.
+	// CoreJS polyfills are automatically dropped using the browserslist definitions; no need to include them here.
+	const polyfillsSkippedInEvergreen = [
+		// Local storage used to throw errors in Safari private mode, but that's no longer the case in Safari >=11.
```

Comment:
What's the new behavior in 11+? Does it drop stored values entirely or does it throw away stored values when the session / page ends?

Reply:
The latter: https://trac.webkit.org/changeset/215315/webkit